hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/debug/ChunkKeyHandler.java

```diff
@@ -102,8 +100,9 @@ protected void execute(OzoneClient client, OzoneAddress address)
       }
       ContainerLayoutVersion containerLayoutVersion = ContainerLayoutVersion
           .getConfiguredVersion(getConf());
-      JsonArray responseArrayList = new JsonArray();
-      for (OmKeyLocationInfo keyLocation : locationInfos) {
+      ArrayNode responseArrayList = JsonUtils.createArrayNode();
+      for (OmKeyLocationInfo keyLocation : keyInfo.getLatestVersionLocations()
+          .getBlocksLatestVersionOnly()) {
```

Comment:
Also seems to be unrelated changed.  Can we keep using `locationInfos`?

```suggestion
      for (OmKeyLocationInfo keyLocation : locationInfos) {
```

Reply:
Done!
Thanks!